src/main/java/com/github/wovnio/wovnjava/Settings.java

```diff
@@ -37,6 +37,11 @@
 
         String p;
 
+        p = config.getInitParameter("userToken");
+        if (p != null && p.length() > 0) {
+            this.projectToken = p;
```

Comment:
if both userToken and projectToken are in configuration, then projectToken should not be replaced. Imo you miss a case in your if statement.